### PR TITLE
Setup docker container for the application

### LINKED ISSUE #43: dockerize application

### PR DESCRIPTION
**Pull request by elaynelemos**
Closes #43 

Hey! This is my contribution as I told at the [issue](https://github.com/bartius-nigel/weather-comparitor/issues/43#issuecomment-699699349).

#### How to analyse this PR
- Build the container with
```bash
docker-compose up -d weather-app
```
- Build and run the application with
```
./run.sh npm install
# served at http://localhost:5000
./run.sh npm run dev
```
- Try some commands with
```
./run.sh [COMMAND]
```

The base container chosen was [node](https://hub.docker.com/_/node):12-buster-slim. You can find instructions also at the README.md file.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '3.3'
+
+services:
+  weather-app:
+    image: node:12-buster-slim
+    container_name: wheater_comparitor_app
```
Comment (bartius-nigel) [inline docker-compose.yml:6]:
```suggestion
    container_name: weather_comparitor_app
```

**Review by bartius-nigel — CHANGES_REQUESTED** [review]
just one minor comment regarding some spelling, but looks good to me. Thanks for the contribution!!

File: docker-compose.yml (hunk 1 above)
Comment (elaynelemos) [reply docker-compose.yml]: Woow, thank you! I just fixed it. :smile_cat:

**Review by bartius-nigel — APPROVED** [review]

**Comment by bartius-nigel** [comment]
Thank you so much @elaynelemos !!

**Comment by elaynelemos** [comment]
> Thank you so much @elaynelemos !!

My pleasure! :smile_cat:

**Pull request merged**